lib/sentry/integrations/oban/cron.ex

```diff
@@ -68,9 +70,19 @@ defmodule Sentry.Integrations.Oban.Cron do
   end
 
   defp job_to_check_in_opts(job) when is_struct(job, Oban.Job) do
+    # lookup job.id -> if exists check-in-id is this uuid. Othereise store new key and create UUID value
     if schedule_opts = schedule_opts(job) do
+      {:ok, id} =
+        case Mapping.lookup(:cron, "oban-#{job.id}") do
+          {:ok, value} ->
+            {:ok, value}
+
+          :error ->
+            Mapping.insert(:cron, "oban-#{job.id}", UUID.uuid4_hex())
+        end
```

Comment:
Hrm, I’m thinking this is not exactly what we want. I think that we want:

  * that if this is a "start" event, then the ID **must not** be there, so we want to insert it and
  * that if this is a "stop" or "exception" event, then the ID **must** be there, so we expect it.

Does that make sense?